.github/workflows/test-pr.yml

```
@@ -107,6 +107,11 @@ jobs:
             test-args:
             timeout: 45
             parallel: 1
+          - name: 'Summarization'
+            test-suite: 'test-summarize'
+            test-args:
+            timeout: 360
```

```suggestion
            timeout: 150
```

We can shorten this noww.